js/views/userPageVw.js

```diff
@@ -450,6 +456,7 @@ module.exports = Backbone.View.extend({
   renderItems: function (model) {
     "use strict";
     var self = this;
+    var select = document.getElementById('categories');
```

Comment:
On second thought, in the events this element is referenced by js-categories, it would be best to be consistent, so this.$el.find('.js-categories')